src/integrationTest/java/integration/tests/SystemEngineDatabaseMetaDataTest.java

```diff
@@ -53,7 +53,14 @@ void readOnly() throws SQLException {
     @Tag("v2")
     void getSchemas() throws SQLException {
         String database = integration.ConnectionInfo.getInstance().getDatabase();
-        assertEquals(List.of(List.of("information_schema", database)), getSchemas(DatabaseMetaData::getSchemas));
+        assertEquals(List.of(List.of("information_schema", database)), getRows(DatabaseMetaData::getSchemas));
+    }
+
+    @Test
+    @Tag("v2")
+    void getCatalogs() throws SQLException {
+        String database = integration.ConnectionInfo.getInstance().getDatabase();
+        assertTrue(getRows(DatabaseMetaData::getCatalogs).contains(List.of(database)));
     }
```

Comment:
This currently does not test the new functionality. We had the previous getCatalogs return a single database in a list.
A way to improve this would be to create another database (it's cheap to do so in Firebolt) by running sql `CREATE DATABASE test_catalogs` and then check that the result list contains both our current database as well as `test_catalogs`. This way we verify the new behaviour.
Just remember to run `DROP DATABASE test_catalogs` afterwards.